src/main/java/org/jboss/tools/intellij/openshift/actions/application/DeleteApplicationAction.java

```diff
@@ -24,27 +24,37 @@
 import java.io.IOException;
 import java.util.concurrent.CompletableFuture;
 
+import static org.jboss.tools.intellij.openshift.telemetry.TelemetryService.TelemetryResult;
+
 public class DeleteApplicationAction extends OdoAction {
-  public DeleteApplicationAction() {
-    super(ApplicationNode.class);
-  }
-
-  @Override
-  public void actionPerformed(AnActionEvent anActionEvent, TreePath path, Object selected, Odo odo) {
-    ApplicationNode applicationNode = (ApplicationNode) selected;
-    NamespaceNode namespaceNode = applicationNode.getParent();
-    if (Messages.NO == Messages.showYesNoDialog("Delete Application '" + applicationNode.getName() + "'.\nAre you sure?", "Delete Application",
-        Messages.getQuestionIcon())) {
-        return;
+
+    public DeleteApplicationAction() {
+        super(ApplicationNode.class);
+    }
+
+    @Override
+    protected String getTelemetryActionName() { return "delete application"; }
```

Comment:
Please use a prefix. IJ Tekton uses 'ui ', better check with @adietish

Reply:
@jeffmaury i've introduced a prefix constant, and choose "ui/", because the action nanmes can contains spaces and "ui-" will look like "ui-delete application". with my proposal, it will be "ui/delete application". @adietish WDYT ? is "/" supported in actions names ?

Reply:
@sbouchet according to woopra "/" would have the same problem as ":": no schema would be generated automatically. The allowed separator-characters are 
* " "
* "-"
* "_"

https://github.com/redhat-developer/intellij-tekton/pull/341#issuecomment-815179517
> When a new event without a schema comes into Woopra, the system will auto-generate a basic schema so that the event can be found in reports and searched for in the Woopra interface. However, for Schemas to be auto-generated, the allowed characters are -- lower case letters, numbers, underscore, dashes, and spaces. Since the actions contain a colon ":" they'll just have to be added to the schema manually since the character will prevent it from being automatically added.

To be fully sure you can send test events and then verify in woopra that schema are generated. I'd guess they're not given what we've been told.